pkg/kube/volumemounts.go

```diff
@@ -41,6 +41,12 @@ func GetVolumeMounts(instance *sfv1alpha1.SplunkForwarder) []corev1.VolumeMount
 			MountPath: "/opt/splunkforwarder/etc/apps/osd_monitored_logs/metadata",
 		},
 
+		// State Mount
```

Comment:
Is there documentation out there that explains that Splunk's state is stored under here? If so it might be good to comment that here.

Reply:
It's poorly documented, but I added a link to https://www.splunk.com/en_us/blog/tips-and-tricks/what-is-this-fishbucket-thing.html